site/en/blog/tablesng/index.md

```diff
@@ -0,0 +1,170 @@
+{% Video src="video/HodOHWjMnbNw56hvNASHWSgZyAf2/QKOKGUUuVP6i86nyDdeH.mp4", autoplay="true", muted="true", loop="true"  %}
+<figcaption>
+The table header is not remaining in the sticky position, despite <code>position: sticky</code> being applied to the <code>&lt;thead&gt;</code>.</figcaption>
+</figure>
+
+The reason we've had this bug around for so long is because `position: sticky`
+was specified well after HTML tables came out. Before this fix, headers with an
+intended `position: sticky` were just [converted
+into](https://github.com/w3c/csswg-drafts/issues/3136) `position: static`, but
+now, you can use `position: sticky` anywhere in the tables: on headers (`<thead>`) or
+vertical axis labels. 
+
+<figure>
+{% Video src="video/HodOHWjMnbNw56hvNASHWSgZyAf2/SZc6cN9UeFhPZ1q2BRyq.mp4", autoplay="true", muted="true", loop="true"  %}
+<figcaption>
+Table header has sticky positioning in Chromium 91+. <a href="https://codepen.io/una/pen/zYZdgMJ">See Demo on
+Codepen</a>.
+</figcaption>
+</figure>
+
+{% Aside 'warning' %} 
+If you're using `position: sticky` on a table, make sure
+it doesn't have borders. Border painting is currently an open cross-browser
+compatibility issue, as borders belong to the table, not the header row itself.
+{% endAside %}
+
+
+
+## Improved border painting and background painting
+
+One of the oldest table
+[bugs](https://bugs.chromium.org/p/chromium/issues/detail?id=2902), which dates
+back to September 2008, was filed almost as soon as Chrome was released, and was
+never able to be fixed due to the old table architecture. The issue surrounds
+table painting and collapsed borders:
+
+The way that tables are painted, in order of `z-index`, is: cells > rows >
+sections > tables. They then are painted by the order they appear in the DOM
+(Document Object Model), though the cells themselves are in reverse DOM order,
+where the first cell in the table is the top-most.
+
+{% Img src="image/HodOHWjMnbNw56hvNASHWSgZyAf2/KsyTCCXLRhF2bdxlZP96.png",
+alt="z-index order of tables", width="800", height="434" %}
+
+So the issue here is that borders belong to the table, not the cell, in the old
+way that tables were painted. Collapsed borders are painted when table paints its foreground. 
+This means that a single table cell could not have
+multiple borders:
+
+<figure>
+{% Img src="image/HodOHWjMnbNw56hvNASHWSgZyAf2/Ms8AqAJn1oKmM1thWYut.png",
+alt="correct and incorrect table rendering", width="800", height="333" %}
+<figcaption>Left: Incorrect table rendering prior to TablesNG. Right: correct display of a table’s borders in TablesNG.</figcaption>
+</figure>
+
+
+In the example above, you can see that the leftmost blue cell was incorrectly
+painting on top of the orange bottom-right cell since it could not have multiple
+borders. In the re-architected implementation, this is solved, and the orange
+border cell correctly paints on top of the blue one, allowing the second table
+gap to have both blue and orange border lines.
```

Comment:
```suggestion
In the example above, you can see that the leftmost blue cell was incorrectly
painting on top of the orange bottom-right cell as it could not have multiple
borders. In the re-architected implementation, this is solved, and the orange
border cell correctly paints on top of the blue one, allowing the second table
gap to have both blue and orange border lines.
```